assets/javascripts/discourse/routes/group-reports-show.js

```diff
@@ -9,15 +9,20 @@ export default DiscourseRoute.extend({
     const group = this.modelFor("group");
     return ajax(`/g/${group.name}/reports/${params.query_id}`)
       .then((response) => {
-        const queryParamInfo = response.query.param_info;
+        let query = response.query;
+        let query_group = response.query_group;
```

Comment:
nit: We use calemCase for variables in JS

Reply:
Noted. Done.